vendor/Ltac2/.gitignore

```diff
@@ -0,0 +1,18 @@
+Makefile.coq
+Makefile.coq.conf
+*.glob
+*.d
+*.d.raw
+*.vio
+*.vo
+*.cm*
+*.annot
+*.spit
+*.spot
+*.o
+*.a
+*.aux
+tests/*.log
+*.install
+_build
+.merlin
```

Comment:
I guess most of this file should go.

Reply:
Yes, that's what I meant by the TODO item above.